api/src/main/java/org/openmrs/module/eptsreports/reporting/cohort/definition/ResumoMensalTransferredOutCohortDefinition.java

```diff
@@ -0,0 +1,43 @@
+package org.openmrs.module.eptsreports.reporting.cohort.definition;
```

Comment:
@ynurmahomed I would recomend that we seat with the team to explain this approach and explore where it best fits for us to use it